framework/graphics/fps_info.cpp

```diff
@@ -26,7 +26,7 @@
 #include "util/date_time.h"
 #include "util/logging.h"
 
-#include <inttypes.h>
+#include <cinttypes>
```

Comment:
I'll put in a comment similar to the one I put before: We use the format macros everywhere with no errors without this #define. Is this specific to a compiler or system headers version?  Can you tell me more about why you had to add this?

Reply:
The reason it works everywhere else is because cinttypes is used everywhere else. So I removed the #define and fixed the #include instead.

Reply:
Sorry, let me try to be more clear.  This **file** compiles on other platforms, including Linux x86.  So why do you need to alter it to compile on ARM linux?

Reply:
This actually works fine on the ARM platforms I've tried. The problem occurs on our x86-based HPC platforms, which can have fairly old toolchains. I believe the root cause is that traditionally, inttypes.h has required __STDC_FORMAT_MACROS to enable these macros for C++, while cinttypes removed this requirement earlier than inttypes.h. See https://cplusplus.github.io/LWG/issue984

If you want to enable using these format macros on the very oldest of toolchains, you should add __STDC_FORMAT_MACROS globally, but that is probably overkill as affected toolchains are now very old. Since changing to cinttypes works for us and makes things more internally consistent in gfxreconstruct, I see this as a reasonable solution in this case.